Ladybird/Qt/BrowserWindow.cpp

```diff
@@ -910,6 +918,31 @@ void BrowserWindow::update_displayed_zoom_level()
     m_current_tab->update_reset_zoom_button();
 }
 
+void BrowserWindow::set_window_rect(Optional<Web::DevicePixels> x, Optional<Web::DevicePixels> y, Optional<Web::DevicePixels> width, Optional<Web::DevicePixels> height)
+{
+    auto current_geometry = geometry();
+    current_geometry.moveTopLeft(mapToGlobal(current_geometry.topLeft()));
+    dbgln("current_geometry: {} {} {} {}", current_geometry.x(), current_geometry.y(), current_geometry.width(), current_geometry.height());
```

Comment:
Looks like a whole bunch of debugging got accidentally committed here

Reply:
Somehow ~~Palpatine returned~~ my changes from my macOS testing didn't get pushed up, should be all fixed now.